README.md

```diff
@@ -76,7 +76,7 @@ $ .ci/build
 To update pre-configured images supported k8s versions run the following commands and then commit the resulting changes:
 ```bash
 gardenctl target garden <landscape>
```

Comment:
There's one more mention of gardenctl here.

Reply:
this one could stay.. it just describes how to update the k8s versions